packages/react-core/src/components/OverflowMenu/OverflowMenuDropdownItem.tsx

```diff
@@ -1,28 +1,25 @@
 import * as React from 'react';
-import { DropdownItem, DropdownItemProps } from '../Dropdown';
+import { DropdownItem, DropdownItemProps } from '../../next/components/Dropdown';
 import { OverflowMenuContext } from './OverflowMenuContext';
 
-export interface OverflowMenuDropdownItemProps extends DropdownItemProps {
+export interface OverflowMenuDropdownItemProps extends Omit<DropdownItemProps, 'ref'> {
   /** Indicates when a dropdown item shows and hides the corresponding list item */
   isShared?: boolean;
-  /** Indicates the index of the list item */
-  index?: number;
 }
 
 export const OverflowMenuDropdownItem: React.FunctionComponent<OverflowMenuDropdownItemProps> = ({
   children,
   isShared = false,
-  index,
   ...additionalProps
 }: OverflowMenuDropdownItemProps) => (
   <OverflowMenuContext.Consumer>
     {value =>
       (!isShared || value.isBelowBreakpoint) && (
-        <DropdownItem component="button" index={index} {...additionalProps}>
+        <DropdownItem component="button" {...additionalProps}>
           {children}
         </DropdownItem>
       )
     }
   </OverflowMenuContext.Consumer>
 );
-OverflowMenuDropdownItem.displayName = 'OverflowMenuDropdownItem';
+OverflowMenuDropdownItem.displayName = 'OverflowMenuItem';
```

Comment:
This should be kept as-is since there's already the OverflowMenuItem component that has this display name.